webapp/src/types/types.ts

```diff
@@ -37,6 +37,7 @@ export type CallsClientConfig = {
     simulcast?: boolean;
     enableAV1: boolean;
     dcSignaling: boolean;
+    dcLocking: boolean;
 }
```

Comment:
I think we talked about this, but it's been awhile I guess -- shouldn't this be moved over to calls-common?  Maybe that was a TODO...?

Reply:
@cpoile This is not RTCPeer, it's the wrapper, which at this point is only used on this side.